examples/ProgModel.cpp

```diff
@@ -56,7 +56,7 @@ void ProgModel::LoadOFF(std::string const & path) {
     mMeshes.push_back(mesh);
 
 #pragma omp parallel for
-    for (size_t i = 0; i < mMeshes.size(); ++i) {
+    for (int i = 0; i < mMeshes.size(); ++i) {
```

Comment:
`error C3016: 'i': index variable in OpenMP 'for' statement must have signed integral type`